src/bioregistry/data/bioregistry.json

```diff
@@ -61183,6 +61183,7 @@
       }
     ],
     "twitter": "MonarchInit",
+    "uri_format": "https://monarchinitiative.org/disease/$1",
```

Comment:
With `uri_format` would `$1` be replaced with `prefix:identifier` or `identifier`?

> All of the OBO Foundry ontologies should get a OBO PURL as their priority URI format

So should we update so we set `uri_format` to produce a URI like `http://purl.obolibrary.org/obo/MONDO_0004992`? Or will this occur automatically once an upstream change is made?

Reply:
Always use the OBO purl, mondo decides itself through the obo purl system where to redirect

Reply:
`$1` is always only the local unique identifier.